tests/acceptance/test_cases/acceptance_test_case.py

```diff
@@ -0,0 +1,55 @@
+"""
+Base class for acceptance tests of the API. Acceptance tests run on the
+entire server stack, and check that TopChef works end-to-end
+"""
+import unittest
+from topchef import APP_FACTORY
```

Comment:
I haven't ended up using the ``TestingAppFactory`` defined in ``wsgi_app`` in the test suite. If we're going to run acceptance tests, then we should take care to use a copy of the production application. For this test suite, we should look towards running as close to a production environment as possible. Therefore, get rid of the ``TestingAppFactory``.